src/pages/vendor-profile/components/vendor-buttons-group/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from "@mui/material";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { StyledButton, StyledVendorsButtonsGroup } from "./styles.ts";
+
+const zIndexZero = 0;
+const zIndexOne = 1;
```

Comment:
Scale theme and create const for this.
Use 0-10 for now, we need decide later, how better.